### PR TITLE
Fix Bug 1214022, additions to the mozilla trademark list

### PR DESCRIPTION
**Pull request by schalkneethling**
@alexgibson r?

### CODE REVIEW

**Comment by schalkneethling** [comment]
Good for review, just waiting for confirmation on the bug. Thanks!

**Comment by alexgibson** [comment]
The new images are twice the size of the others, I would suggest sizing them to be consistent?

trademarks

File: bedrock/foundation/templates/foundation/trademarks/list.html [hunk 1]
```diff
@@ -19,9 +19,15 @@ <h1 class="page-title">{{ _('List of Mozilla Trademarks') }}</h1>
       <th>{{ _('Firefox logo') }}&reg;</th>
     </tr>
     <tr>
+      <td>
+        {{ high_res_img('trademarks/hello-logo-wordmark.png', { 'alt': 'Firefox Hello logo' }) }}
```
Comment (alexgibson) [inline bedrock/foundation/templates/foundation/trademarks/list.html]: The `alt` text should be wrapped for l10n here and below.

Comment (schalkneethling) [inline bedrock/foundation/templates/foundation/trademarks/list.html]: Yes, I was wondering whether this page is localized. I am guessing it is? If so, this probably needs an l10n label as a heads up to @flodolo

Comment (alexgibson) [inline bedrock/foundation/templates/foundation/trademarks/list.html]: I don't believe it is localized (but @flodolo can confirm). Either way, existing strings are already wrapped on this page so we should still wrap new ones.

**Comment by schalkneethling** [comment]
@alexgibson Yes, the size of the logos and whether or not it should be just the logo or, logo and wordmark, is what I am waiting confirmation on. With that said, I agree with your suggestion and will make the update. Thanks!

**Comment by alexgibson** [comment]
@schalkneethling - if we are waiting on possible changes based on feedback in the bug, is it best to wait on review here if the code is likely to be changing again?

File: bedrock/foundation/templates/foundation/trademarks/list.html (hunk 1 above)
Comment (flodolo) [inline bedrock/foundation/templates/foundation/trademarks/list.html]: No, not localized (but good thing in general to have strings wrapped).

**Comment by schalkneethling** [comment]
@alexgibson nope, the code won't change. The only possible change are the images and possibly their size.

**Comment by alexgibson** [comment]
Please let me know when changes are made and this is good to go, and I will check again :+1:

**Comment by alexgibson** [comment]
@schalkneethling - any update on this one yet, is it good for review / merge-able?

**Comment by schalkneethling** [comment]
@alexgibson Looks like it might be, just waiting on final confirmation from Jen.

**Comment by schalkneethling** [comment]
@alexgibson Ok, this is good for review and merge. Thank you

File: bedrock/foundation/templates/foundation/trademarks/list.html [hunk 2]
```diff
@@ -15,127 +15,113 @@ <h1 class="page-title">{{ _('List of Mozilla Trademarks') }}</h1>
...
-      <th>{{ _('Camino') }}&reg;</th>
+      <td><img width="100" height="100" alt="{{ _('SeaMonkey logo') }}®" src="{{ static('img/trademarks/seamonkey-logo-r.png') }}" /><span>&reg;</span></td>
+      <th>{{ _('SeaMonkey logo') }}</th>
     </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Mozilla Communities') }}™" src="{{ static('img/trademarks/community-wordmark.png') }}" /></td>
-
-      <th>{{ _('Mozilla Communities') }}&trade;</th>
+      <td><img width="100" height="100" alt="{{ _('Foxkeh logo') }}®" src="{{ static('img/trademarks/foxkeh-logo-r.png') }}" /><span>&reg;</span></td>
+      <th>{% trans %}Foxkeh logo<br /><small>This trademark is licensed under the Creative Commons Attribution-Non-Commercial-NoDerivs 2.1 Japan.</small>{% endtrans %}</th>
```
Comment (alexgibson) [inline bedrock/foundation/templates/foundation/trademarks/list.html]: Might this be better as two separate strings rather than using a trans block?

**Comment by alexgibson** [comment]
One minor nit, but otherwise looks good r+wc :turtle:

File: bedrock/foundation/templates/foundation/trademarks/list.html (hunk 2 above)
Comment (schalkneethling) [inline bedrock/foundation/templates/foundation/trademarks/list.html]: That would be better. I did not change it and it is as it was before I believe but, I like your suggestion and, this is not currently translated so, will not be breaking l10n

**Pull request merged**